lib/geolocation.js

```diff
@@ -53,8 +53,9 @@ async function setLocationWithIdb (idb, latitude, longitude) {
     } catch (e) {
       throw new Error(`Failed to set geolocation with idb. Original error: ${e.stderr || e.message}`);
     }
+  } else {
```

Comment:
I think it would be more readable to turn this into a guard and move to the beginning of the method:

```
if (!idb) {
  throw ...
}
```

Reply:
Agreed